R/utils.R

```diff
@@ -59,7 +59,9 @@ get_tests_from_lintr <- function(name) {
   )
   dest <- paste0("tests/testthat/test-", name, ".R")
   utils::download.file(url, destfile = dest)
-  file.edit(dest)
+  if (!is_testing()) {
```

Comment:
I promise this is the last comment for today (or forever on this PR) but I wonder whether it'd make sense to use `rlang::is_interactive()` instead.

Reply:
Right, that should be more robust.

> I promise this is the last comment for today

Keep them coming, you'll get a nice backlog of posts to write 😄

Reply:
https://masalmon.eu/2025/06/25/open-files-folders-websites/ :smile_cat: